### PR TITLE
Initial deploy

### PR DESCRIPTION
**Pull request by dylangood**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM node:boron
+
+# Create app directory
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+# Install app dependencies
+COPY package.json /usr/src/app/
+RUN npm install
```
Comment (gbaik) [inline Dockerfile]: Is there a reason why we are using npm install instead of yarn install?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+COPY . /usr/src/app
+
+EXPOSE 8080
```
Comment (gbaik) [inline Dockerfile]: If this is port do we have to do process.env.port thing?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,15 @@
+FROM node:boron
+
+# Create app directory
+RUN mkdir -p /usr/src/app
```
Comment (gbaik) [inline Dockerfile:4]: So would it be /usr/src/app or path.join(__dirname + 'usr/src/app') ? Same thing for the other ones?

File: Dockerfile (hunk 2 above)
Comment (shihaohong) [reply Dockerfile]: I agree with Gideon, i think you need a conditional to use process.env if it's going to be hosted ona  server

**Pull request closed without merging**